### DISCUSSION TITLE
Unable to get started with "Cannot inline bytecode built with JVM target 11 into bytecode that is being built with JVM target 1.8" error

### DISCUSSION THREAD

**Discussion by AShynkevich** [korlibs/korge#2089] [Q&A]
Hello, 
I tried to create "hello world" application on Korge engine.

OS: MacOS Sonoma 14.1.1
IDE: IntelliJ IDEA Ultimate
Java SDK: 17

I go over steps:
1. I create "Hello world" (all steps by default)
<img width="561" alt="image" src="https://github.com/korlibs/korge/assets/2409272/ddaff0bd-1ac9-4567-b35f-163a014380f6">

2. Waiting for all dependencies are downloaded and indexed
3. Once all necessary things were indexed - code looks good.
4. Then close the project and open it again an see the problem below:
but have the following trouble in the code within IDE(see the picture):
<img width="1097" alt="image" src="https://github.com/korlibs/korge/assets/2409272/027c50bf-bb56-4093-a2d7-de358b58cd0b">
5. The IDE writes "Cannot inline bytecode built with JVM target 11 into bytecode that is being built with JVM target 1.8. Please specify proper '-jvm-target' option".

This error stacks me to proceed any progress with the Engine.

Could you assist me to resolve this?

P.S. The related topic: https://github.com/korlibs/korge/issues/2028 doesn't help anyhow

**Comment by soywiz** [comment]
I have moved this into an issue here: https://github.com/korlibs/korge/issues/2094 so we can investigate.
